core/arch/arm/plat-ls/platform_config.h

```diff
@@ -29,6 +28,11 @@
 #ifndef PLATFORM_CONFIG_H
 #define PLATFORM_CONFIG_H
 
+#ifndef ASM
+#include <types_ext.h>
+void get_hw_unique_key(uint64_t smc_func_id, uint64_t in_key, uint64_t size);
+#endif
+
```

Comment:
Move this to `core/arch/arm/plat-ls/main.c` because this function is not supposed to be called from anywhere else if I'm not mistaken

Reply:
Ok.